cpu/atmega_common/Makefile.dep

```diff
@@ -0,0 +1,19 @@
+ifneq (,$(filter atmega_pcint,$(USEMODULE)))
+    USEMODULE += atmega_pcint0 atmega_pcint1 atmega_pcint2 atmega_pcint3
```

Comment:
This will trigger the `#error` in the c file. (I know it is a pitty that it now needs to be handled twice, but I believe the error message can save a lot of time for end users and developers of this feature.)

Reply:
Don't quite get the comment / problem here? Can you specify your comment?

Reply:
Oooh. You mean using only atmega_pcint also triggers the atmega_pcint3 error...